CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### New features
+
+* [#277](https://github.com/rubocop/rubocop-rails/pull/277): Add new `Rails/ForeignKeyName` cop. ([@fatkodima][])
```

Comment:
Would you please change to the new changelog format? https://github.com/rubocop/rubocop-rails/commit/3cae05d8cf1a94e76496a42bf38e033f941690f6#diff-a66abd3d7020d7e9f081637dbc85308d8cd44db8e7eec508b9314f3664ff6968